### PR TITLE
Refactoring File_P2_Clip

### PR DESCRIPTION
**Pull request by pavel-pimenov**

### CODE REVIEW

File: Source/MediaInfo/Multiple/File_P2_Clip.h [hunk 1]
```diff
@@ -47,7 +46,7 @@ private :
 
     //Buffer - File header
     bool FileHeader_Begin();
-
```
Comment (JeromeMartinez) [inline Source/MediaInfo/Multiple/File_P2_Clip.h]: For next time: try to avoid unecessary diff e.g. empty line removal (here and above too).

Comment (pavel-pimenov) [inline Source/MediaInfo/Multiple/File_P2_Clip.h]: ok. Recreate PR ?

Comment (JeromeMartinez) [inline Source/MediaInfo/Multiple/File_P2_Clip.h]: just update the PR with old commit removed and new commit (but if you don't know how to do it, I may accept a new PR)

**Comment by pavel-pimenov** [comment]
If that's right, I'll create another PR

**Comment by pavel-pimenov** [comment]
I learned git + squash :)

C:\git\MediaInfoLib>git rebase -i HEAD~2
[detached HEAD 8e1ebca] Refactoring File_P2_Clip
 Date: Fri Dec 30 13:50:19 2016 +0300
 2 files changed, 24 insertions(+), 53 deletions(-)
Successfully rebased and updated refs/heads/fix-File_P2_Clip.

C:\git\MediaInfoLib>git push origin fix-File_P2_Clip -f
Counting objects: 7, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (7/7), done.
Writing objects: 100% (7/7), 1.02 KiB | 0 bytes/s, done.
Total 7 (delta 6), reused 0 (delta 0)
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
To https://github.com/pavel-pimenov/MediaInfoLib.git
 + 07f2059...8e1ebca fix-File_P2_Clip -> fix-File_P2_Clip (forced update)

**Pull request merged**